plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -213,7 +216,23 @@ public SqlServerClient(
         this.statisticsEnabled = statisticsConfig.isEnabled();
 
         this.connectorExpressionRewriter = JdbcConnectorExpressionRewriterBuilder.newBuilder()
+                .add(new RewriteUnicodeVarcharConstant())
```

Comment:
I put it first, as far as I remember @wendigo explained me that only first one will be executed, because we have in next raw similar rule (during debug I saw that really only this one executed)

Reply:
deserves a comment about why this rule exists - since it doesn't exist anywhere else.